wfe2/wfe.go

```diff
@@ -1250,3 +1253,40 @@ func (wfe *WebFrontEndImpl) NewOrder(ctx context.Context, logEvent *requestEvent
 		return
 	}
 }
+
+// Order is used to retrieve a existing order object
+func (wfe *WebFrontEndImpl) Order(ctx context.Context, logEvent *requestEvent, response http.ResponseWriter, request *http.Request) {
+	id, err := strconv.ParseInt(request.URL.Path, 10, 64)
+	if err != nil {
+		wfe.sendError(response, logEvent, probs.Malformed("Invalid ID"), err)
```

Comment:
Nit: can you make the `Malformed` detail `"Invalid order ID"` to help disambiguate "ID"?